### DISCUSSION TITLE
能否实现英雄彩蛋语音

### DISCUSSION THREAD

**Discussion by Boquirroto** [Pik-4/HsMod#200] [Ideas (许愿池)]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

发现换皮后使用铁定卡牌触发的彩蛋语音还是原皮的

### Describe the solution you'd like/描述你想要的解决方案

是否可以换皮后能有新皮的彩蛋语音

**Comment by Pik-4** [comment]
能否列举出几个样例（比如友方是什么英雄，打出什么卡牌，触发什么语音），我对于彩蛋不是很了解

**Comment by Boquirroto** [comment]
> 能否列举出几个样例（比如友方是什么英雄，打出什么卡牌，触发什么语音），我对于彩蛋不是很了解

比如艾萨拉女王打出‘奥术智慧’触发‘我的智慧无以富加’，还有使用奥弹、法反、镜像、炎爆也有彩蛋语音。基本每个皮肤都有使用彩蛋，一般都会有四五个。之前阿凯换皮萨兰德使用远古谜团的时候显示的是萨兰德的彩蛋语音，所以我问能不能实现一下这个功能。

**Comment by irememoryi** [comment]
是啊，不能触发替换的皮肤使用特定卡牌的语音，就是位于炉石文件夹data\Win的unity包里带有'Trigger'tag的语音,其他语音都是正常的，可能炉石代码里对于这种特定卡牌的触发类语音读取方式不一样？不懂....希望大佬能够实现一下这个功能

**Comment by irememoryi** [comment]
这个功能已经实现了,不知道大佬什么时候完善的
